tests/app/questionnaire/routing/test_operators.py

```diff
@@ -343,5 +369,27 @@ def test_nonetype_operands_for_array_operators(operator_name, operands):
     ],
 )
 def test_operation_count(operands, expected_result):
-    operator = Operator(Operator.COUNT)
+    operation = get_operation(Operator.COUNT)
+    operator = Operator(Operator.COUNT, operation)
     assert operator.evaluate(operands) is expected_result
+
+
+def get_operation(operator):
```

Comment:
This is a pain, but as operations will eventually be initialized with schema, locale etc and the function names don't match up I have had to put a map here, however I realise this is duplicated in when_rule_evaluator.